akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -462,6 +468,8 @@ class HttpExt private[http] (private val config: Config)(implicit val system: Ex
     val hostHeader = if (port == connectionContext.defaultPort) Host(host) else Host(host, port)
     val layer = clientLayer(hostHeader, settings, log)
     layer.joinMat(_outgoingTlsConnectionLayer(host, port, settings, connectionContext, log))(Keep.right)
+      // already added in clientLayer but needed here again to also include transport layer
+      .addAttributes(cancellationStrategyAttributeForDelay(settings.streamCancellationDelay))
```

Comment:
since this is `(clientLayer.join(tlsLayer)).addAttributes(delay)` that also covers the inner one, should it only be here then?

Reply:
That's right, it also covers the inner one but the idea is, that if you use `clientLayer` alone, it should still be applied. Here TCP and TLS are added which would otherwise be missing it.

Reply:
Ah, that is public API so users could be using it, missed that.